app/assets/css/sass/_album.scss

```diff
@@ -27,4 +27,12 @@
 	.rtmedia-media {
 		width: 100%;
 	}
+}
+
+.rtmedia-album-list {
+	.rtmedia-list-item {
+		border: 1px solid #ddd !important;
```

Comment:
Could we avoid usage of `! important here`? Try adding more [specificity](https://dev.to/emmawedekind/css-specificity-1kca) and see if that helps in removing `! important`.